addons/account/tests/common.py

```diff
@@ -928,6 +934,7 @@ def _jsonify_company(self, company):
         return {
             'id': company.id,
             'tax_calculation_rounding_method': company.tax_calculation_rounding_method,
+            'account_fiscal_country_id': self._jsonify_country(company.account_fiscal_country_id),
```

Comment:
I think what you want is `web_read`
For instance
```python
In [1]: self.env['account.move'].browse(1).web_read({
   ...:     'name': {},
   ...:     'line_ids': {'fields': {
   ...:         'account_id': {'fields': {
   ...:             'display_name': {},
   ...:         }}
   ...:     }}
   ...: })
Out[1]: 
[{'id': 1,
  'name': 'INV/2025/00001',
  'line_ids': [{'id': 5,
    'account_id': {'id': 27, 'display_name': '400000 Product Sales'}},
   {'id': 6, 'account_id': {'id': 27, 'display_name': '400000 Product Sales'}},
   {'id': 27, 'account_id': {'id': 22, 'display_name': '251000 Tax Received'}},
   {'id': 40,
    'account_id': {'id': 7, 'display_name': '121000 Account Receivable'}}]}]

```

Reply:
Interesting. I'll think about it

Reply:
@william-andre I'll try this way in another PR/commit if I found some time to do that